x-pack/plugins/cases/common/types/api/case/v1.test.ts

```diff
@@ -259,6 +290,26 @@ describe('Status', () => {
         PathReporter.report(CasePostRequestRt.decode({ ...defaultRequest, category }))
       ).toContain('The length of the category is too long. The maximum length is 50.');
     });
+
+    it('removes foo:bar attributes from customFields', () => {
```

Comment:
I think we should check for extra fields inside the `field` property. Checking one `type` should be fine.